src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -273,12 +273,14 @@
             :last-item-style style/last-community-tag
             :container-style style/community-tag-container}])
         [join-community community pending?]]
-       [channel-list-component
-        {:on-category-layout              on-category-layout
-         :community-id                    id
-         :community-color                 color
-         :on-first-channel-height-changed on-first-channel-height-changed}
-        (add-handlers-to-categorized-chats id chats-by-category joined)]])))
+       (when (or (and (seq token-permissions) joined)
+                 (not (seq token-permissions)))
```

Comment:
`(not (seq` can be replaced by `empty?` https://github.com/clojure/clojure/blob/clojure-1.11.1/src/clj/clojure/core.clj#L6246